src/xb-node.c

```diff
@@ -365,6 +389,70 @@ xb_node_get_attr_as_uint (XbNode *self, const gchar *name)
 	return g_ascii_strtoull (tmp, NULL, 10);
 }
 
+/**
+ * xb_node_attr_iter_init:
+ * @iter: an uninitialized #XbNodeIter
+ * @self: a #XbNode
+ *
+ * Initializes a name/value pair iterator for the node attributes
+ * and associates it with @self.
+ *
+ * Since: 0.3.4
+ **/
```

Comment:
```suggestion
 */
```

Finishing a doc comment with a double-star is long deprecated.

Reply:
Ohh? Looks like I need to fix up all my projects then! Is there any doc I can reference for the commit message?

Reply:
Me too, I also had no idea. But the double-star also make the documentation block look a bit more special than a regular, non-doc comment...

Reply:
> Ohh? Looks like I need to fix up all my projects then! Is there any doc I can reference for the commit message?

I don’t think it’s worth fixing old instances of, but is just about worth ensuring new instances are right. I don’t have a reference which mentions the change, but [the examples in the gtk-doc manual](https://developer-old.gnome.org/gtk-doc-manual/stable/documenting_syntax.html.en) all use `*/`.

It’s really not important